<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISF Face Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ImageWidget" name="image" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionOpenCamera"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpenCamera">
   <property name="icon">
    <iconset resource="FaceViewer.qrc">
     <normaloff>:/res/camera.svg</normaloff>:/res/camera.svg</iconset>
   </property>
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="FaceViewer.qrc">
     <normaloff>:/res/disconnect.svg</normaloff>:/res/disconnect.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close Image Source</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="FaceViewer.qrc">
     <normaloff>:/res/about.svg</normaloff>:/res/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="FaceViewer.qrc">
     <normaloff>:/res/image.svg</normaloff>:/res/image.svg</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="FaceViewer.qrc"/>
 </resources>
 <connections/>
</ui>
